shardingsphere-scaling/shardingsphere-scaling-bootstrap/src/test/java/org/apache/shardingsphere/scaling/web/HttpServerInitializerTest.java

```diff
@@ -45,9 +42,8 @@
     private ChannelPipeline channelPipeline;
     
     @Before
-    @SneakyThrows(ReflectiveOperationException.class)
     public void setUp() {
-        ReflectionUtil.setFieldValue(ScalingContext.getInstance(), "serverConfig", new ServerConfiguration());
+        ServerConfigurationInitializer.init();
```

Comment:
This change seem no relate with your PR, why to change it?

Reply:
Because in my local test, I found that the serverConfig is empty. When instantiating the HttpServerHandler, the checkServerConfig method can’t get through.